lib/SILGen/SILGenDistributed.cpp

```diff
@@ -274,6 +274,34 @@ void InitializeDistActorIdentity::dump(SILGenFunction &) const {
 #endif
 }
 
+bool SILGenFunction::shouldReplaceConstantForApplyWithDistributedThunk(
```

Comment:
Nit: Can this be moved to `Callee` and renamed into something like `shouldDispatchWitnessViaDistributedThunk` seems like that's the only place where we'd actually want to do something about it, `SGF` is available there and plays a contextual role in the check which is actually about the function being called.

Reply:
Sounds good, moving! Thank you

Reply:
Done, thanks!